README.md

```diff
@@ -198,3 +198,92 @@ func Function(
 	
 }
 ```
+
+### Sync Tool and Operator Types
+
+Previously, packages relied on importing the types from individual operators and adding them as dependencies. This has led to eco-goinfra having many dependencies, making it harder to reuse and increasing the chance of version conflicts between indirect dependencies.
+
+By using just the runtime client and removing dependencies on individual operators, the sync tool reduces the number of eco-goinfra dependencies and makes packages more modular. It will periodically copy over types from operator repos to this repo by cloning the desired path in the operator repo, applying changes in the config file, then copying it into this repo.
+
+#### Running
+
+To run the sync tool, use the `lib-sync` makefile target.
+
+```
+make lib-sync
+```
+
+It may be faster to sync only one config file instead using the `--config-file` flag.
+
+```
+go run ./internal/sync --config-file ./internal/sync/configs/<config-file.yaml>
+```
+
+If the sync fails while adding a new set of operator types, remove the synced directory from `schemes/` and rerun the sync.
```

Comment:
Just to be safe I would add an example placeholder i.e. "remove the synced directory from `schemes/<pkg-to-sync>` and rerun the sync."